chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -36,11 +38,10 @@
   EOH
```

Comment:
Just wondering: is this still correct? Shouldn't we call `update-ca-certificates` instead?